modules/install-sno-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,167 @@
+<8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
+<9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+
+. Generate {product-title} assets by running the following commands:
++
+[source,terminal]
+----
+$ mkdir ocp
+----
++
+[source,terminal]
+----
+$ cp install-config.yaml ocp
+----
++
+[source,terminal]
+----
+$ ./openshift-install --dir=ocp create single-node-ignition-config
+----
+
+. Obtain the {op-system-base} `kernel`, `initramfs`, and `rootfs` files from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or from the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/latest/[{op-system} image mirror] page.
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the highest version that is less than or equal
+to the {product-title} version that you install.Only use the appropriate `kernel`, `initramfs`, and `rootfs` artifacts described in the following procedure.
+====
++
+The file names contain the {product-title} version number. They resemble the following examples:
+
+** kernel: `rhcos-<version>-live-kernel-<architecture>`
+** initramfs: `rhcos-<version>-live-initramfs.<architecture>.img`
+** rootfs: `rhcos-<version>-live-rootfs.<architecture>.img`
++
+. Move the downloaded {op-system-base} live `kernel`, `initramfs`, and `rootfs` as well as the Ignition files to an HTTP or HTTPS server before you launch `virt-install`.
+
+. Create the new KVM guest nodes using the {op-system-base} `kernel`, `initramfs`, and Ignition files, the new disk image, and adjusted parm line arguments.
++
+[source,terminal]
+----
+$ virt-install \
+   --name {vn_name} \
+   --autostart \
+   --memory={memory_mb} \
+   --cpu host \
+   --vcpus {vcpus} \
+   --location {media_location},kernel={rhcos_kernel},initrd={rhcos_initrd} \ <1>
+   --disk size=100 \
+   --network network={virt_network_parm} \
+   --graphics none \
+   --noautoconsole \
+   --extra-args "ip=${IP}::${GATEWAY}:${MASK}:${VM_NAME}::none" \
+   --extra-args "nameserver=${NAME_SERVER}" \
+   --extra-args "ip=dhcp rd.neednet=1 ignition.platform.id=metal ignition.firstboot" \
+   --extra-args "coreos.live.rootfs_url={rhcos_liveos}" \ <2>
+   --extra-args "ignition.config.url={rhcos_ign}" \ <3>
+   --extra-args "random.trust_cpu=on rd.luks.options=discard" \
+   --extra-args "console=tty1 console=ttyS1,115200n8" \
+   --wait
+----
+<1> For `--location`, specify the location of the kernel/initrd on the HTTP or HTTPS server.
+<2> For `coreos.live.rootfs_url=`, specify the matching rootfs artifact for the `kernel` and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<3> For `ignition.config.url=`, specify the Ignition file for the machine role. Only HTTP and HTTPS protocols are supported.
```

Comment:
The literals need to be followed by a noun, such as the `--location` flag, `kernel` and `initramfs` artifacts, or `coreos.live.rootfs_url=` parameter.